lib/webidl2.js

```diff
@@ -429,7 +489,11 @@
           return { type: "sequence", value: [] };
         } else {
           const str = consume(STR) || error("No value for default");
-          str.value = str.value.replace(/^"/, "").replace(/"$/, "");
+          if (!str.value.startsWith('"'))
+            error(`string '${str.value}' doesn't start with a quote`);
+          if (!str.value.endsWith('"'))
+            error(`string '${str.value}' doesn't end with a quote`);
```

Comment:
Hmm, this cannot happen as the regex forces them, right? We have AST tests, so probably no need to do runtime check here. What do you think? @marcoscaceres

Reply:
I don't think so. I just left it there in case there was something I was missing.